### PR TITLE
fix: Settings: Hide Credentials Menu

### PR DESCRIPTION
**Pull request by MeenuyD**
Commented out Credentials menu item and hidden corresponding page.
#9042

### CODE REVIEW

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thank you, @MeenuyD 👍 

Will you please also address this lint error? https://github.com/meshery/meshery/actions/runs/6444472783/job/17497500170?pr=9045#step:4:1883

**Comment by MeenuyD** [comment]
can you tell me how can fix it

**Comment by leecalcote** [comment]
@MeenuyD, yes, read line 1882 - https://github.com/meshery/meshery/actions/runs/6444472783/job/17497500170?pr=9045#step:4:1882

**Comment by sandramsc** [comment]
@MeenuyD please familiarize yourself with the [Contributing guidelines](https://layer5.io/community/newcomers): you need to request for an issue to be assigned to you first before opening PR (this reduces the risk of wasting your time and effort as a result of the issue being rightfully assigned to someone else), i.e only after it is assigned to you then open the PR. On this occasion since nobody has requested for the issue to be assigned to them, please comment on the issue asking it to be assigned to you and I'll do this.

**Comment by MeenuyD** [comment]
Ok thanks for this.

**Comment by sudhanshutech** [comment]
@MeenuyD it is a good practice to ask for assigning to work on issue so it will not create conflict between other contributors

File: ui/components/MesherySettings.js [hunk 1]
```diff
@@ -393,14 +393,14 @@ class MesherySettings extends React.Component {
                 tab="registry"
               />
             </Tooltip>
-            <Tooltip title="Credential" placement="top">
+            {/* <Tooltip title="Credential" placement="top">
```
Comment (sudhanshutech) [inline ui/components/MesherySettings.js:396]: @MeenuyD the eslint check will fail due to trailing space, use `//` to comment out the code for now

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by sudhanshutech** [comment]
@MeenuyD dco check is failing, this is because you are signing of commit, do use `git commit -s`or see this [DOC](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits)

File: ui/components/MesherySettings.js (hunk 1 above)
Comment (MeenuyD) [reply ui/components/MesherySettings.js:396]: can you please explain it more?

**Comment by leecalcote** [comment]
Resyncing branch

**Comment by abhijeetgauravm** [comment]
Let's discuss this on Meshery Development Meeting today at 8:30 PM IST / 9 AM Central time. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#).

**Comment by Yashsharma1911** [comment]
Thankyou for raising your hand on this task, this issue is taken care of in #9152, closing this PR, I'll encourage you to join meshery dev meeting and pick a similar kinds of issues, lemme know if you need help

**Comment by MeenuyD** [comment]
Ok

**Pull request closed without merging**